tests/test_feature_selection.py

```diff
@@ -0,0 +1,101 @@
+# Authors: Soledad Galli <solegalli1@gmail.com>
+# License: BSD 3 clause
+import pytest
+import pandas as pd
+
+from feature_engine.feature_selection import DropFeatures
+
+
+def test_FeatureEliminator_initial(dataframe_vartypes, dataframe_na):
+    # test case 1: with dataframe_vartypes
+    transformer = DropFeatures(features_to_drop=["City", "dob"])
+    X = transformer.fit_transform(dataframe_vartypes)
+
+    # expected result
+    df = pd.DataFrame(
+        {'Name': ['tom', 'nick', 'krish', 'jack'],
+         'Age': [20, 21, 19, 18],
+         'Marks': [0.9, 0.8, 0.7, 0.6]}
+    )
+
+    # init params
+    assert transformer.features == ['City', 'dob']
+    # transform params
+    assert X.shape == (4, 3)
+    assert type(X) == pd.DataFrame
+    pd.testing.assert_frame_equal(X, df)
+
+
+def test_FeatureEliminator_non_existing_variables(dataframe_vartypes):
+    # test case 2: passing variables that doesn't exist
```

Comment:
nice name. Could you please replace by DropFeautres? And could you do this for all the tests below?